src/main/java/racingcar/domain/Cars.java

```diff
@@ -41,12 +49,12 @@ public List<Car> getCurrentStatus() {
 
     public List<String> getWinners() {
         int maxMoveCount = cars.stream()
-                .mapToInt(Car::getMoveCount)
+                .mapToInt(Car::getPosition)
                 .max()
                 .orElse(0);
 
         return cars.stream()
-                .filter(car -> car.getMoveCount() == maxMoveCount)
+                .filter(car -> car.getPosition() == maxMoveCount)
```

Comment:
소소한 의견입니다만 꺼내서 비교하기 보단 객체에게 물어보는 방식이 
더 좋을 것 같습니다 😄 
```suggestion
                .filter(car -> car.isMatchPosition(maxMoveCount))
```

디미터 법칙이 있는데요
같이 정리하고 가시면 좋을 것 같습니다 🙇 
> https://dundung.tistory.com/203